### ISSUE TITLE
[BUG] @stylexswc/postcss-plugin logs a warning for default import source @stylexjs/stylex

### ISSUE THREAD

**Issue by welkinwong** [Dwlad90/stylex-swc-plugin#880]
### Describe the bug

Hi, I noticed what looks like a small bug in `@stylexswc/postcss-plugin`.
When `importSources` contains the default StyleX package name `@stylexjs/stylex`, the plugin logs this warning during development:

```text
SyntaxError: Unexpected token '@', "@stylexjs/stylex" is not valid JSON
From the implementation, it looks like string entries in importSources are being passed to JSON.parse():
```

```js
parsedImportSources = importSources?.map(importSource => {
  return typeof importSource === 'string' ? JSON.parse(importSource) : importSource;
});
```
But @stylexjs/stylex is a normal module specifier, not JSON, so this seems incorrect.

The app still works because the error is caught and the plugin falls back to the original value, so this does not appear to be a fatal build issue. However, it produces misleading warning output in dev logs.

Environment
@stylexswc/postcss-plugin: 0.15.2
@stylexswc/rs-compiler: 0.15.2
@stylexjs/stylex: 0.18.2

Thanks!

### Error stack

```
[Rspack Client Error] SyntaxError: Unexpected token '@', "@stylexjs/stylex" is not valid JSON
    at JSON.parse (<anonymous>)
    at /Users/welkin/code/superdocs-next/node_modules/@stylexswc/postcss-plugin/dist/bundler.js:18:64
    at Array.map (<anonymous>)
    at Object.shouldTransform (/Users/welkin/code/superdocs-next/node_modules/@stylexswc/postcss-plugin/dist/bundler.js:17:50)
    at /Users/welkin/code/superdocs-next/node_modules/@stylexswc/postcss-plugin/dist/builder.js:210:26
    at Array.forEach (<anonymous>)
    at Object.build (/Users/welkin/code/superdocs-next/node_modules/@stylexswc/postcss-plugin/dist/builder.js:207:26)
    at plugins (/Users/welkin/code/superdocs-next/node_modules/@stylexswc/postcss-plugin/dist/plugin.js:96:41)
    at LazyResult.runOnRoot (/Users/welkin/code/superdocs-next/node_modules/postcss/lib/lazy-result.js:361:16)
    at LazyResult.runAsync (/Users/welkin/code/superdocs-next/node_modules/postcss/lib/lazy-result.js:290:26)
    at LazyResult.async (/Users/welkin/code/superdocs-next/node_modules/postcss/lib/lazy-result.js:192:30)
    at LazyResult.then (/Users/welkin/code/superdocs-next/node_modules/postcss/lib/lazy-result.js:436:17)
```

### OS

macOS

### Browser

Chrome 146.0.7680.165

### Compiler version

@stylexswc/rs-compiler: 0.15.2 (installed as a transitive dependency of `@stylexswc/postcss-plugin`)

### Bundler plugin(s) version

@stylexswc/postcss-plugin: 0.15.2  @stylexswc/unplugin": "^0.15.2

### To Reproduce

1. Install `@stylexswc/postcss-plugin@0.15.2`, `@stylexswc/unplugin@0.15.2`, `@stylexswc/rs-compiler@0.15.2`, and `@stylexjs/stylex@0.18.2`
2. Configure `@stylexswc/postcss-plugin` in `postcss.config.mjs` without overriding `importSources`
3. Add a file that imports StyleX with `import * as stylex from '@stylexjs/stylex'`
4. Start the dev server
5. Check the terminal output

The following warning is printed:
```text
SyntaxError: Unexpected token '@', "@stylexjs/stylex" is not valid JSON
```

### Additional information

- [x] I have checked with the latest version

**Comment by Dwlad90** [comment]
Hey @welkinwong,
Thanks for finding the bug.

The fix is available in [0.15.3-rc.2](https://github.com/Dwlad90/stylex-swc-plugin/releases/tag/0.15.3-rc.2).
